src/applications/disability-benefits/996/containers/IntroductionPage.jsx

```diff
@@ -200,6 +201,7 @@ export class IntroductionPage extends React.Component {
                   onClick={() => {
                     this.setWizardStatus(WIZARD_STATUS_NOT_STARTED);
                     this.setPageFocus();
+                    recordEvent({ event: `howToWizard-start-over` });
```

Comment:
These backticks can probably just be quotes.

Reply:
LOL thanks, I'll do that in a different PR.. hate to wait another 30 min just for backticks.